internal/oci/runtime_oci.go

```diff
@@ -400,23 +400,26 @@ func (r *runtimeOCI) ExecSyncContainer(c *Container, command []string, timeout i
 		os.RemoveAll(logPath)
 	}()
 
-	var args []string
-	args = append(args,
+	args := []string{
 		"-c", c.id,
 		"-n", c.name,
 		"-r", r.path,
 		"-p", pidFile,
-		"-e")
+		"-e",
+		"-l", logPath,
+		"--socket-dir-path", r.config.ContainerAttachSocketDir,
+		"--log-level", logrus.GetLevel().String(),
```

Comment:
nit, to appease my ocd, please consider alphabetizing these options.

Reply:
addressed